utility/src/main/java/org/oppia/util/parser/UrlImageParser.kt

```diff
@@ -0,0 +1,81 @@
+package org.oppia.util.parser
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.graphics.Canvas
+import android.graphics.Rect
+import android.graphics.drawable.BitmapDrawable
+import android.graphics.drawable.Drawable
+import android.text.Html
+import android.widget.TextView
+import com.bumptech.glide.request.target.SimpleTarget
+import com.bumptech.glide.request.transition.Transition
+import org.oppia.util.R
+import javax.inject.Inject
+import javax.inject.Singleton
+
+// TODO (#169) : Replace this with exploration asset downloader
```

Comment:
`// TODO (#169): Replace this with exploration asset downloader`

Reply:
Actually:

``// TODO(#169): Replace this with exploration asset downloader.``

(The 'TODO' and '(' as well as the ')' and ':' should be close together).

Reply:
corrected.